### ISSUE TITLE
Initial MVP prototyping

### ISSUE THREAD

**Issue by RenataKeck** [department-of-veterans-affairs/va.gov-team#69497]
Update 11/29: Using Sketch for the time being. Flow should be landing page, creating a new account signup through Login.gov, signing in and getting to the dashboard.

[Link to Sketch file.](https://www.sketch.com/s/e1d6898a-a709-4a4f-a09b-ea0ac2a960a2)

- [x] Obtain sketch access @uxrebecca 
- [x] Pull wireframes for Auth/Login.gov (refer to [auth flow context](https://docs.google.com/document/d/1gQOPIwLmuIMrDXvYEE08XjV3IQfh3jBEEbNN7KNIIJw/edit) doc)
- [x] Landing page @uxrebecca 
- [x] Dashboard @RenataKeck

**Comment by uxrebecca** [comment]
I created an [Accredited Rep Facing project](https://www.sketch.com/workspace/c99d3e96-7c23-4210-b173-92a2b73a8788/p/42aa8cab-86f9-4387-91bb-b5c32e1dcefc) and an [Accredited rep login space](https://sketch.com/s/e1d6898a-a709-4a4f-a09b-ea0ac2a960a2) for us to start working on. I've been exploring the pattern library components but haven't been able to find any components/patterns for login.gov and authentication. Does this exist? I asked in the #benefits-accredited-rep-design Slack channel.

**Comment by uxrebecca** [comment]
The landing page is ready to share for discussion. We ran out of time in our design sync with ARM last week, but said we'd get async feedback over Slack this week so I posted there. Also talked it over with Lesley, and she thought it was great for us to start talking through our assumptions and what will be most useful. She suggested adding content related to getting accredited and managing your accreditation (e.g. getting re-accredited, updating your fee schedule if you're an attorney) - this is managed by OGC.
